profiles/FlimsyLC - PREP v1.0.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Flimsy LC - PREP",
```

Comment:
Pls pay attention to the [profile creation guidelines](https://github.com/Zer0-bit/gaggiuino/tree/community/profiles#creating-a-profile)